README.md

```diff
@@ -219,9 +219,18 @@ pip install ably==2.0.0b3
 ```python
 from ably import AblyRealtime
 
+
+# Create a client using an Ably API key
```

Comment:
This code snippet looks a bit weird now because there are two separate import statements. I think probably the best approach to documenting this is to to have two separate code snippets, one for creating a client with an API key string, and another for creating a client using kwargs.

I also think we should mention the other possible auth options so the comment should look something like this (paraphrasing the ably-js readme)
```
# Create a client using kwargs, which must contain at least one auth option
# the available auth options are key, token, token_details, auth_url, and auth_callback
# see https://www.ably.com/docs/rest/usage#client-options for more details
```